react-ui/src/pages/entry/Entry.jsx

```diff
@@ -152,6 +159,9 @@ function Entry() {
       case 'r':
         redditCommentButtonRef.current.click();
         return;
+      case '0':
+        // clear vote?
```

Comment:
0 is a valid vote, so we can't use that to clear the vote.

Reply:
Changed to 'c'... did not intend to commit that lol